static/js/settings_config.ts

```diff
@@ -123,11 +123,11 @@ export const email_address_visibility_values = {
         code: 1,
         description: $t({defaultMessage: "Admins, moderators, members and guests"}),
     },
-    // // Backend support for this configuration is not available yet.
-    // admins_and_members: {
-    //     code: 2,
-    //     description: $t({defaultMessage: "Members and admins"}),
-    // },
+    // Backend support for this configuration is not available yet.
+    members: {
+        code: 2,
+        description: $t({defaultMessage: "Admins, moderators and members"}),
+    },
     moderators: {
         code: 5,
         description: $t({defaultMessage: "Admins and moderators"}),
```

Comment:
Would it be a good idea to align with the numbers for given groups used in `common_policy_values`? Or I guess `invite_to_realm_policy_values`, as the one with a `nobody` value? I think that to do that, we'd mostly just need to use the more standard values in `models.py` and map the IDs a bit differently in the migration file...

I guess the main question is whether that sort of alignment might make our life a bit easier when migrating to this being a "groups" based setting in the future, but maybe it doesn't matter and it's better to not mess with things... and I think the answer is no, in that we're just going to map a given value onto a given group using the `EMAIL_ADDRESS_VISIBILITY_EVERYONE` type names? But I'll leave it to you to decide -- we only have two settings today that don't share the meanings of which `code` goes with which group with the `common_policy_values` structure -- this one and `wildcard_mention_policy` -- and this is a basically free opportunity to make this one work the same as all the others.

As a side note, there's the question of whether we want to be migrating to a role dropdown rather than a "pick a group" setting here. I think the answer is probably we should merge this as written, because the conditionals of the form "Is this EMAIL_ADDRESS_VISIBILITY_EVERYONE" will be hard to port without doing a database query, and that seems like enough of a rabbit hole that we're probably better off having that be an independent migration in the future. But wanted to raise this in case you disagree, since you're probably going to be the one doing such a migration down the line.

Reply:
Regarding the option values, I think we would just require to map the values in migration file and I don't think it would matter much there for what values we have for this and other settings.

And I think it's better to do this as role thing and not groups. We still have some things to figure out for groups based settings regarding code structure and this PR is already big enough.

Reply:
Yeah, I agree on both points, resolving.